packages/address/CHANGELOG.md

```diff
@@ -5,7 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-<!-- ## Unreleased -->
+## Unreleased
+
+### Added
+
+- Exposes formatting functions publicly so they can be used synchronously [#1843](https://github.com/Shopify/quilt/pull/1843) [#2178](https://github.com/Shopify/quilt/pull/2178)
```

Comment:
A consumer will read "new functions exposed" and will then immediately think "neat, what are they" - let's just tell them rather than making them hunt:

```suggestion
- Exposes `formatAddress`, `buildOrderedFields` formatting functions publicly so they can be used synchronously [#1843](https://github.com/Shopify/quilt/pull/1843) [#2178](https://github.com/Shopify/quilt/pull/2178)
```